clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -455,18 +459,15 @@ export class CopybookDownloadService implements vscode.Disposable {
   }
 
   public async start() {
-    let startTime: number | null = null;
+    const startTime = Date.now();
```

Comment:
Are you sure if this equivalent change?
```
await this.queue.pop();
```
looks like blocking call

Reply:
await this.queue.pop(); can take some time(), that will mess up time measurement.
clients/cobol-lsp-vscode-extension/src/services/copybook/DownloadQueue.ts:59 and clients/cobol-lsp-vscode-extension/src/services/copybook/DownloadQueue.ts:83 if I reed it correctly